assets/knative/service.yaml.tmpl

```diff
@@ -8,3 +8,5 @@ spec:
     spec:
       containers:
         - image: {{ .RemoteTag }}
+          ports:
+          - containerPort: {{ .Port }}
```

Comment:
This should only be added if port is defined

Reply:
```suggestion
{{ if .Port }}
          ports:
          - containerPort: {{ .Port }}
{{ end }}
```

Reply:
Fixed